web/src/components/ui/tag-selector.tsx

```diff
@@ -36,192 +16,120 @@ interface TagsSelectFormFieldProps
+                      onClick={(event) => {
+                        setSelectedValues([]);
+                        onValueChange([]);
+                        event.stopPropagation();
                       }}
-                      className="cursor-pointer"
-                    >
-                      <span>{option}</span>
-                    </CommandItem>
-                  );
-                })}
-              </CommandGroup>
-              <CommandSeparator />
-              <CommandGroup>
-                <div className="flex items-center justify-between">
-                  {selectedValues.length > 0 && (
-                    <>
-                      <CommandItem
-                        onSelect={() => {
-                          setSelectedValues([]);
-                          selectedValuesSet.current.clear();
-                          onValueChange([]);
-                        }}
-                        style={{
-                          pointerEvents: "auto",
-                          opacity: 1,
-                        }}
-                        className="flex-1 justify-center cursor-pointer"
-                      >
-                        Clear
-                      </CommandItem>
-                      <Separator
-                        orientation="vertical"
-                        className="flex min-h-6 h-full"
-                      />
-                    </>
-                  )}
-                  <CommandSeparator />
-                  <CommandItem
-                    onSelect={() => setIsPopoverOpen(false)}
-                    style={{
-                      pointerEvents: "auto",
-                      opacity: 1,
-                    }}
-                    className="flex-1 justify-center cursor-pointer"
-                  >
-                    Close
-                  </CommandItem>
-                </div>
-              </CommandGroup>
-            </CommandList>
-          </Command>
-        </PopoverContent>
-      </Popover>
-    );
-  }
-);
+                    />
+                    <Separator orientation='vertical' className='flex min-h-6 h-full' />
+                  </>
+                )}
+                <Popover.Button>
+                  <ChevronDown className='h-4 mx-2 cursor-pointer text-muted-foreground' />
+                </Popover.Button>
+              </div>
+            </Popover.Button>
+
+            <Popover.Panel className='w-[240px] flex flex-col gap-4 z-50 w-72  rounded-md border bg-popover p-4 text-popover-foreground shadow-md outline-none data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0 data-[state=closed]:zoom-out-95 data-[state=open]:zoom-in-95 data-[side=bottom]:slide-in-from-top-2 data-[side=left]:slide-in-from-right-2 data-[side=right]:slide-in-from-left-2 data-[side=top]:slide-in-from-bottom-2 p-0 drop-shadow-sm absolute z-10  '>
```

Comment:
cleanup classes since it has multiple `z-*`, `p-*` also `w-[240px]` and `w-72` are the same